.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Report a bug encountered with the ingress2gateway project
+labels: kind/bug
+
+---
+<!--
+Thank you for your interest in Ingress2Gateway! Please note that bug reports 
+here should only be used for bugs with the API itself, such as:
+
+- Incomplete or inaccurate validation
+- Problems with the API specification
+- Unexpected errors with the validating webhook
+- Flaws in conformance tests
+
+Please use this template while reporting a bug and provide as much info as
+possible. Not doing so may result in your bug not being addressed in a timely
+manner. Thank you!
```

Comment:
Most of this description does not feel relevant for ingress2gateway, and is really only intended for the main gateway-api repo.

Reply:
Thanks, update them later.